README.md

```diff
@@ -0,0 +1,24 @@
+
+# Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) with the ODH Notebook Controller as the launcher.
+
+## Container Image Layering
+
+```mermaid
+
+flowchart TD
+
+ubi8/python-3.8 --> Base(base-ubi8-py38);
+
+```
```

Comment:
We want to use the same image name everywhere (base-ubi8-python-3.8), like this:

```mermaid
graph TB
    subgraph Main Tree
    %% base
    ubi8-python-3.8(UBI8 Python 3.8)-->base-ubi8-python-3.8
    base-ubi8-python-3.8("Notebooks base<br/>(base-ubi8-python-3.8)")
    end
```